tests/functional/s3transfer/test_crt.py

```diff
@@ -156,6 +160,26 @@ def _assert_exected_s3express_request(
             awscrt.auth.AwsSigningAlgorithm.V4_S3EXPRESS
         )
 
+    def _assert_exected_mrap_request(
```

Comment:
nit:

```suggestion
    def _assert_expected_mrap_request(
```

This is a copy paste from other mis-named tests, so fine to clean this up at another time.

Reply:
I'll do a quick follow up PR cleaning up everything so we don't lose it if one of these needs to get reverted.

Reply:
https://github.com/aws/aws-cli/pull/9087